11/alpine/Dockerfile

```diff
@@ -4,3 +4,13 @@ RUN apk add --no-cache \
     bash \
     ca-certificates \
     curl
+
+# Installs latest Chromium (73) package.
```

Comment:
can you do it in the previous step please ?

better install all the deps at once ;) (and create a unique docker layer for that)